src/customers/dto/create-customer.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateCustomerDTO {
+  @ApiProperty({ description: 'The first name of the customer' })
+  firstName: string;
```

Comment:
```
{
    "firstName": null,
    "lastName": "Spring",
    "username": "JavaScript5",
    "password": "youllneverguess2",
    "emailAddress": "email@address.com"
}
```
If we use the DTO from above it can cause errors. We should make sure this is not the case!

For DTOs we can use some `class-validator` to make sure that we allow a correct payload into the app. This was not in the training details. Please implement something like this for only one DTO if you want to see it how it works. Consider it low priority.